src/content/docs/it/install/manual.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Installa Astro manualmente
+description: Come installare Astro manualmente con NPM, PNPM, o Yarn.
+---
+import Button from '~/components/Button.astro'
+import FileTree from '~/components/FileTree.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+Questa guida ti mostrerà come installare e configurare un nuovo progetto Astro se preferisci non usare [lo strumento CLI automatico](/it/install/auto/).
+
+#### Preferisci un modo più rapido per iniziare?
+
+<Button href="/it/install/auto/">Prova lo strumento CLI **create astro** →</Button>
+
+#### Prerequisiti
+
+- **Node.js** - `v16.12.0` o superiore.
+- **Editor di testo** - Raccomandiamo [VS Code](https://code.visualstudio.com/) con la nostra [Estensione Ufficiale Astro](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
+- **Terminale** - Astro viene utilizzato tramite la sua interfaccia da riga di comando (CLI).
+
+#### Installazione
+
+Se preferisi non usare il nostro strumento CLI automatico `create astro`, puoi impostare tu stesso il tuo progetto seguendo questa guida.
+
+## 1. Crea la tua cartella
+
+Crea una cartella vuota con il nome del tuo progetto, e naviga al suo interno.
+
+```bash
+mkdir il-mio-progetto-astro
+cd il-mio-progetto-astro
+```
+
+Una volta all'interno della cartella, crea il file `package.json` del tuo progetto. Questo servirà a gestire le dipendenze del tuo progetto, incluso Astro. Se questo formato di file non ti è familiare, esegui il seguente comando per crearne uno.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm init --yes
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm init 
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn init --yes
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+
+## 2. Installa Astro
+
+Per prima cosa, installa le dipendenze di un progetto Astro all'interno del tuo progetto.
+
+:::note[Importante]
+Astro deve essere installato localmente, non globalmente. Assicurati di *non* star eseguendo `npm install -g astro`, `pnpm add -g astro` oppure `yarn add global astro`.
+:::
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install astro
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm install astro 
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Poi, sostituisci qualunque sezione segnaposto "scripts" all'interno del tuo file `package.json` con il seguente codice:
```

Comment:
"segnaposto"? It seems out of place 🤔

Reply:
Yeah, it's not the most elegant solution but I was going for a literal translation. I'll think of a better one as soon as I get back home.